docs/book/v1/intro.md

```diff
@@ -75,26 +104,18 @@ Basic Access Authentication* adapter and the *htpasswd* file as the user
 repository.
 
 ```php
-// ConfigProvider.php
-
-use Zend\Expressive\Authentication\AuthenticationInterface;
-use Zend\Expressive\Authentication\UserRepositoryInterface;
-
-class ConfigProvider
-{
+// src/ConfigProvider.php
```

Comment:
Same comment here; do this as a dependency configuration array, not part of a `ConfigProvider`.